shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui25/components/NowLabel.kt

```diff
@@ -0,0 +1,42 @@
+package org.jetbrains.kotlinconf.ui25.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+import org.jetbrains.compose.ui.tooling.preview.Preview
+import org.jetbrains.kotlinconf.ui25.theme.KotlinConfTheme
+import org.jetbrains.kotlinconf.ui25.theme.PreviewHelper
+
+
+@Composable
+fun NowLabel(modifier: Modifier = Modifier) {
+    Row(verticalAlignment = Alignment.CenterVertically) {
+        Box(
+            Modifier.size(10.dp)
+                .clip(CircleShape)
+                .background(KotlinConfTheme.colors.accentText)
+        )
+        Spacer(Modifier.size(4.dp))
+        KCText(
+            text = "Now",
+            color = KotlinConfTheme.colors.accentText,
+            style = KotlinConfTheme.typography.text2
+        )
+    }
+}
+
+@Composable
+@Preview
+fun NowLabelPreview() {
```

Comment:
please consider making previews private, so they will not display in completion

Reply:
Didn't realize they can be private. Fixed!